### ISSUE TITLE
High level tutorial with key steps 

### ISSUE THREAD

**Issue by grandtiger** [mkoertgen/hello.squirrel#4]
Hi Marcel, how are you doing? Thanks for your change to support overriding the version last year! This project has been great, I am able to test out the auto update feature of squirrel.windows and am very happy about how it works. 

However, when I tried to use it in a brand new project, I have hard time getting it to work, even with this working example side-by-side. I checked your [blog](http://mkoertgen.blogspot.com/2015/04/squirrelwindows-starterkit.html), but it doesn't have much details. 

I wonder if you could summarize the key steps we need to follow to get a brand new project up and running. Thanks a lot!

**Comment by mkoertgen** [comment]
Hi @grandtiger, thx.

Well the basic steps to getting started with [Squirrel.Windows](https://github.com/Squirrel/Squirrel.Windows) are documented in its docs, see the [docs overview here](https://github.com/Squirrel/Squirrel.Windows/blob/master/docs/readme.md).

I summarize:
-  `Install-Package squirrel.windows`
- Add `Package.nuspec` to your project with contents like [this](https://github.com/mkoertgen/hello.squirrel/blob/master/HelloSquirrel/Package.nuspec) - documented [here](https://github.com/Squirrel/Squirrel.Windows/blob/master/docs/getting-started/2-packaging.md)

This starterkit just adds some automation on top, i.e.
- [OneClickBuild](https://github.com/awesome-inc/OneClickBuild) which gives you a simple command line build, including automated versioning with [GitVersion](https://github.com/GitTools/GitVersion) and NuGet packaging
- the custom [HelloSquirrel.targets](https://github.com/mkoertgen/hello.squirrel/blob/master/HelloSquirrel/HelloSquirrel.targets) which automated the squirrel packaging and deployment steps. 

Note that the custom targets-files is implicitly imported when using OneClickBuild. However, since NuGet 3 there are no solution level packages so you need to follow some manual steps after 1st time installation, see [here](https://github.com/awesome-inc/OneClickBuild/blob/develop/ReadMe.md#getting-started)

**Comment by mkoertgen** [comment]
I am aware of some recent changes & advancements on Squirrel.Windows, which is cool! 

The current version is [1.2.3](https://www.nuget.org/packages/squirrel.windows/1.2.3) whereas the starterkit depends on 1.2.1.

So looking at the version there should be no breaking changes. Let me check.

**Comment by grandtiger** [comment]
Hi @mkoertgen , thanks a lot for the tutorial and the pointers! I am following the Squirrel.Windows docs to do it manually, and I got the auto update to work. However, there is a very annoying unhandled exception of type 'System.Threading.AbandonedMutexException' occurred in Squirrel.dll, with additional information: Leaked a Mutex! I wonder if you have ever seen this before. I copied most of the auto update code from your example, perhaps I missed something. I am checking right now.
